terraform/application/database.tf

```diff
@@ -36,7 +36,7 @@ module "postgres" {
   azure_enable_monitoring        = var.enable_monitoring
   azure_enable_backup_storage    = var.enable_postgres_backup_storage
   server_version                 = "14"
-  azure_extensions               = ["btree_gin", "citext", "plpgsql", "pg_trgm"]
+  azure_extensions               = ["btree_gin", "citext", "pg_trgm"]
```

Comment:
do we also need to remove the line from `db/schema.rb`? - it currently contains `enable_extension "plpgsql"`

Reply:
we also have `plpgsql` below in the `postgres-snapshot`

Reply:
The current assumption is that Microsoft have trusted `plpgsql` so it doesn't need to be whitelisted and added manually, so this _should_ work. But we'll soon find out 🙈

Reply:
Ah totally missed it down there, well spotted!